exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -121,6 +121,7 @@ func (c *metricsConsumer) Consume(ctx context.Context, md pmetric.Metrics) error
 				} else if !c.config.AppTagsExcluded {
 					resAttrsMap = appAttributesToTags(resAttrs)
 				}
+				fixServiceTag(resAttrsMap)
```

Comment:
Is this call to fixServiceTag necessary?  When handling the various types of metrics, you call pointAndResAttrsToTagsAndFixSource() (For example lines 240, 340) which also seems to call fixServiceTag.

Reply:
You are right. Removing it.